package.json

```diff
@@ -48,7 +48,7 @@
   },
   "resolutions": {
     "@nx/devkit": "17.0.3",
-    "@types/react": "18.2.35",
+    "@types/react": "18.2.45",
```

Comment:
This was causing a warning inside the example app so I've upgraded this across the board.